package.json

```diff
@@ -36,7 +36,9 @@
     "build:test": "cp envConfig/test.env .env && NODE_ENV=production webpack",
     "build:test:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/test.env > .env && NODE_ENV=production webpack",
     "cypress": "cypress run --config-file cypress-simorgh.json",
-    "cypress:onefile": "CYPRESS_APP_ENV=live CYPRESS_SMOKE=false cypress run --config-file cypress-simorgh.json --spec '/Applications/Code/simorgh/cypress/integration/pages/storyPage/*'",
+    "cypress:onefile": "CYPRESS_APP_ENV=live CYPRESS_SMOKE=false cypress run --config-file cypress-simorgh.json --spec './cypress/integration/pages/storyPage/*'",
+    "cypress:oneservice": "CYPRESS_APP_ENV=live CYPRESS_SMOKE=false CYPRESS_ONLY_SERVICE=mundo cypress run --config-file cypress-simorgh.json",
+    "cypress:oneserviceandonefile": "CYPRESS_APP_ENV=live CYPRESS_SMOKE=false CYPRESS_ONLY_SERVICE=mundo cypress run --config-file cypress-simorgh.json --spec './cypress/integration/pages/storyPage/*'",
```

Comment:
Is this list of script getting a bit big? Pretty harmless but just wondering if these were more intended for debugging?

Reply:
no! 

They were really to speed up certain testing and so knowledge isn't lost. 

if you just want to do one service - e.g. just test mundo - here's the command; if you want to do one folder of tests - here's the command, if you want to do one folder _and_ one service (fastest!) then do that. 

Helps devs run targeted tests quickly.

Maybe we should write this up or demo it?